validator/src/commands/run/args.rs

```diff
@@ -1698,18 +1729,52 @@ pub fn add_args<'a>(app: App<'a, 'a>, default_args: &'a DefaultArgs) -> App<'a,
     )
 }
 
+fn validators_set(
+    identity_pubkey: &Pubkey,
+    matches: &ArgMatches<'_>,
+    matches_name: &str,
+    arg_name: &str,
+) -> Result<Option<HashSet<Pubkey>>> {
```

Comment:
Is this a copy/paste of the other function ? If so, can we avoid the duplication ?